doc/api/next_api_changes/removals/21980-CC.rst

```diff
@@ -0,0 +1,4 @@
+Removed logging
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think this is going to fail the docs for too many decorator markers.

Reply:
I was wondering why the doc building failed. Thanks!